components/screens/LoginScreen.js

```diff
@@ -0,0 +1,237 @@
+                        }}
+                    />
+
+                    <LinearTextGradient
+                        style={{
+                            fontSize: 25,
+                            fontWeight: "700",
+                            alignSelf: "center"
+                        }}
+                        colors={[
+                            "#E54500CF",
+                            "#FF5C00C9",
+                            "#FF9921"
+                        ]}
+                        locations={[0, 0.5, 1]}
+                        start={{ x: 1, y: 0 }}
+                        end={{ x: 0, y: 0 }}
+                    >
+                        <Text>Spoon</Text>
+                    </LinearTextGradient>
+                </LinearGradient>
+            </ImageBackground>
+            <View style={styles.loginButtonsContainer}>
+                <TouchableOpacity style={styles.loginButton}>
+                    <Icon
+                        name="google"
+                        size={15}
+                        color="red"
+                        style={styles.loginButtonIcons}
+                    />
+                    <View style={styles.LoginButtonTextContainer}>
+                        <Text style={styles.loginButtonText}>
+                            Continue with Google
+                        </Text>
+                    </View>
+                </TouchableOpacity>
+                <TouchableOpacity style={styles.loginButton}>
+                    <Icon
+                        name="facebook"
+                        size={12}
+                        color="blue"
+                        style={styles.loginButtonIcons}
+                    />
+                    <View style={styles.LoginButtonTextContainer}>
+                        <Text style={styles.loginButtonText}>
+                            Continue with Facebook
+                        </Text>
+                    </View>
+                </TouchableOpacity>
+            </View>
+            <View style={styles.lineSegment}>
+                <Text style={styles.lineSegmentText}>
+                    Or
+                </Text>
+            </View>
+            <View style={styles.loginInputsContainer}>
+                <Text style={styles.loginInputText}>
+                    Email
+                </Text>
+                <CustomInput
+                    placeholderText={"Enter your email"}
+                    inputColor={"white"}
+                    textColor={"#979797"}
+                />
+            </View>
+            <View style={styles.loginInputsContainer}>
+                <Text style={styles.loginInputText}>
+                    Password
+                </Text>
+                <CustomInput
+                    placeholderText={"Enter your password"}
+                    inputColor={"white"}
+                    textColor={"#979797"}
+                />
+            </View>
+            <View style={styles.recoverPasswordContainer}>
```

Comment:
this should be touchable opacity even if we are not going to really have this feature